lib/platform/linux-i2c.c

```diff
@@ -228,41 +244,134 @@ static uint8_t i2c_gas_cap_get(struct switchtec_dev *dev)
 		return (rx_buf[0] & GAS_TWI_MODE_MASK);
 }
 
-/* One I2C transaction can write a maximum of 28 bytes */
-#define I2C_MAX_WRITE 28
+/* One I2C transaction can write a maximum of 26 bytes */
+#define I2C_MAX_WRITE 24
```

Comment:
The difference between the comment and the define is confusing. Please explain in the comment.